api.ts

```diff
@@ -1162,6 +1301,25 @@ export function makeSyncTableLegacy<
   return makeSyncTable({name, identityName: '', schema, formula, connectionRequirement, dynamicOptions});
 }
 
+/**
+ * Creates a dynamic sync table definition.
+ *
+ * @example
+ * ```
+ * coda.makeDynamicSyncTable({
```

Comment:
In a world where we promote the `pack.addThing()` paradigm over the manifest, is there any utility for the `makeThing()` functions?

Reply:
No and yes. If people exclusively use the builder, then we don't need these. If people still want to have an out so that they don't use the builder, we would still want these, they provide some boilerplate and default values that are useful. Right now the builder methods just call these. I think that's fine if that continues to be the case, there aren't meant to be too many of these, just one per building block.